egs/chime4/s5_1ch/local/chime4_train_rnnlms.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
```

Comment:
I can do it starting tomorrow (Wednesday) evening.

On Tuesday, April 26, 2016, Vijayaditya Peddinti <notifications@github.com>
wrote:

> In egs/chime4/s5_1ch/local/chime4_train_rnnlms.sh
> <https://github.com/kaldi-asr/kaldi/pull/730#discussion_r61200604>:
>
> > @@ -0,0 +1,111 @@
> > +#!/bin/bash
>
> @jtrmal <https://github.com/jtrmal> @hainan-xv
> <https://github.com/hainan-xv> would either of you have time to go
> through this file ?
> Add a line note
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/730/files/1bb204ec1d3cadd1d7760d9c6bb84d2cb5dd3357#r61200604>
>


-- 
Sent from Gmail Mobile